pkg/java/pom/utils.go

```diff
@@ -9,3 +12,10 @@ func isDirectory(path string) (bool, error) {
 	}
 	return fileInfo.IsDir(), err
 }
+
+func versionIsProperty(version string) bool {
```

Comment:
We may want to use this function in other places than values.

```suggestion
func isProperty(s string) bool {
```

Reply:
It make sense. Changed in https://github.com/aquasecurity/go-dep-parser/pull/235/commits/cf34368af151f49d98ae63e6a94da3a3b72c89f6.